source-code/app/src/main/java/org/buildmlearn/toolkit/fragment/DraftsFragment.java

```diff
@@ -397,4 +408,22 @@ private void deleteAllDrafts() {
     private void restoreSelectedView() {
         restoreColorScheme();
     }
+
+    @Override
+    public void onBackPressed() {
+
+        if (mAdapter.selectedPositionsSize() != 0){
+            for(int i=0;i<mAdapter.getCount();i++)
+                if(mAdapter.isPositionSelected(i)) {
+                    mListView.getChildAt(i).setBackgroundColor(0);
+                    mAdapter.removeSelectedPosition(i);
+                }
+            restoreColorScheme();
+        }
+        else{
+            FragmentManager fragmentManager = getFragmentManager();
+            fragmentManager.beginTransaction().setTransition(FragmentTransaction.TRANSIT_FRAGMENT_FADE)
```

Comment:
We are no longer using a stack to keep track of fragments after PR #206.
Also the `super.onBackPressed()` method cannot be called in a fragment because `onBackPressed()` cannot be declared as a static method.
Also from PR #206 onwards we are performing a transaction which you can see in the `onBackPressed()` of `HomeActivity.java`.

@opticod 
Your reviews.